frontends/mit-open/webpack.config.js

```diff
@@ -28,7 +28,10 @@ const getFeatureFlags = () => {
 }
 
 const getPostHogSettings = () => {
-  if (process.env.POSTHOG_ENABLED && process.env.POSTHOG_PROJECT_API_KEY) {
+  if (
+    process.env.POSTHOG_ENABLED === "True" &&
```

Comment:
Would it make sense to coerce this such that any non-falsey ( !== '',0,False,false) value means True? Even though it is explicit in this PR what the value should be I could see this causing confusion in the future.

Reply:
Updated in https://github.com/mitodl/mit-open/pull/746/commits/04e60505ac34327809533a5dca425ddea003890b to do similar - it now lowers and trims and checks for "true". (This is pretty similar to how the Django app does it, except it'll raise an exception if it's not either True or False; I don't see a need to be that exacting about it.)